Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebViewModel.swift

```diff
@@ -73,18 +68,23 @@ class PayPalWebViewModel: ObservableObject, PayPalWebCheckoutDelegate {
                     let payPalRequest = PayPalWebCheckoutRequest(orderID: orderID, fundingSource: funding)
                     payPalWebCheckoutClient.start(request: payPalRequest)
                 }
+                updateState(.success)
             } catch {
                 print("Error starting PayPalWebCheckoutClient")
-                state = .error(message: error.localizedDescription)
+                updateState(.error(message: error.localizedDescription))
             }
         }
     }
 
-    func getPayPalClient() async throws -> PayPalWebCheckoutClient {
+    func getPayPalClient() async throws -> PayPalWebCheckoutClient? {
         do {
             let config = try await configManager.getCoreConfig()
             let payPalClient = PayPalWebCheckoutClient(config: config)
             return payPalClient
+        } catch {
+            updateState(.error(message: error.localizedDescription))
```

Comment:
Updated this logic since we wouldn't display anything if we didn't get a config back, no we will return our error view in this case